.github/scripts/trymerge.py

```diff
@@ -1413,7 +1530,17 @@ def categorize_checks(
         if check_runs[checkname].status is None:
             pending_checks.append((checkname, check_runs[checkname].url))
         elif not is_passing_status(check_runs[checkname].status):
-            failed_checks.append((checkname, check_runs[checkname].url))
+            if check_runs[checkname].classification in ('BROKEN_TRUNK', 'FLAKY'):
+                ok_failed_checks.append((checkname, check_runs[checkname].url))
+            else:
+                failed_checks.append((checkname, check_runs[checkname].url))
+
+    if ok_failed_checks:
+        print("The following checks failed but were ok to fail: " + ", ".join([x[0] for x in ok_failed_checks]))
+
+    if len(ok_failed_checks) > ok_failed_checks_threshold:
+        failed_checks = failed_checks + ok_failed_checks
```

Comment:
May be also print something here to know that there are more `failed_checks` than the allow thresholds.  Mainly to make it less confusing after seeing the above log about `The following checks failed but were ok to fail ...`, then now the merge is blocked because of them